boards/posix/native_posix/doc/board.rst

```diff
@@ -628,3 +628,26 @@ can feed it the following set of commands through a pipe:
    echo -e \
    'select kernel\nuptime\n!wait 500\nuptime\n!wait 1000\nuptime\n!quit' \
    | zephyr/zephyr.exe
+
+Host based flash access
+***********************
+
+In case a flash device is present the file system partitions on the flash device
```

Comment:
Shouldn't this commit then change the previous section about flash access, to mention this is also an option?